sdk/nodejs/api/client.gen.ts

```diff
@@ -171,7 +191,16 @@ export type DirectoryEntriesOpts = {
 }
 
 export type DirectoryWithDirectoryOpts = {
+  /**
+   * Exclude artifacts that match the given pattern.
+   * (e.g. ["node_modules/", ".git*"])
```

Comment:
You can also use `@example` here to improve docs readability. (https://typedoc.org/tags/example/)

Reply:
The format is `@example` with a text block stopped by an empty comment line?
The source we use is the graphql schema and uses [graphql description](http://spec.graphql.org/draft/#sec-Descriptions). It doesn't support specific `@example`, neither python nor Go. It would just make life for node people more integrated. And since no other integration exists for other languages, and it's intuitive enough I could be OK for that.
But I would put that in another issue/PR. Can you create one @slumbering ?

Reply:
More info about the format here : https://tsdoc.org/pages/tags/example/
The example above would be like:
/**
   * Exclude artifacts that match the given pattern.
   * @example
   * ["node_modules/", ".git*"]
*/
   
and will be displayed like this:
<img width="863" alt="Capture d’écran 2023-01-04 à 14 37 48" src="https://user-images.githubusercontent.com/1186424/210567137-2076022b-ddd7-4afe-ba34-5bd92bccbb13.png">

Reply:
Issue create @dolanor : https://github.com/dagger/dagger/issues/4323